lib/ansible/modules/find.py

```diff
@@ -149,6 +149,12 @@
             - Default is unlimited depth.
         type: int
         version_added: "2.6"
+    encoding:
+        description:
+            - When doing a C(contains) search, determine the encoding of the files to be searched.
+        type: str
+        default: None
```

Comment:
The equivalent to the None in the argument spec is `null` or `~` in YAML. But you can omit the default.
```suggestion
```